examples/testing/src/main.rs

```diff
@@ -291,8 +291,9 @@ async fn interaction(
         interaction
             .edit_response(
                 ctx,
-                EditInteractionResponse::new()
-                    .new_attachment(CreateAttachment::url(ctx, IMAGE_URL_2).await?),
+                EditInteractionResponse::new().attachments(
+                    EditAttachments::new().add(CreateAttachment::url(ctx, IMAGE_URL_2).await?),
+                ),
```

Comment:
This is a bit wordy. Do you think some of the now-deprecated builder methods could instead be turned into wrappers around `.attachments`? That way, complexity is still centralized into `EditAttachments` but convenience methods are still left. I'd nominate `new_attachment` be kept at least.

Reply:
Good point. Maybe I can actually just un-deprecate all those methods, while carefully documenting what `EditAttachments` invocation they're equivalent to, so users can read about and grasp the unintuitive side effects of those methods.

Reply:
This should be reverted now that `new_attachment` is no longer deprecated.